documents/operating.md

```diff
@@ -11,12 +12,83 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* "**Associate**" shall mean any person legally associated with the Company through contract or ownership.
+* The "**Hive Mind**" shall mean the "board of directors" of the Company, as defined by Pennsylvania state law. The Hive Mind shall consist of all current Corporate Overlords of the Company.
+* "**Plotting Session**" shall mean a meeting of the "board of directors" of the Company, as defined by Pennsylvania state law.
+* "**Quorum**" shall mean the presence of 2/3 or more of the company's Corporate Overlords.
+* "Operating Agreement" shall refer to the official version of this document
+to be held in a file named `operating.md` in the `documents` directory of the
+`corporate` repository within the `BadIdeaFactory` team hosted on GitHub.
+
+### Support Categories
+* "**Complete Consensus**" shall mean 100% support or more.
+* "**Strong Consensus**" shall mean 2/3 support or more.
+* "**Majority**" shall mean over 1/2 support.
+* "**Apathetic Support**" shall mean over 1/3 vocalized support and less than 1/3 vocalized dissent.
```

Comment:
I can kind of understand the notion of `Apathetic Support` as a category for "the org doesn't substantially object", but where practically is this going to be used (it's not referenced anywhere else in the pull requests).